fuzzing_test.go

```diff
@@ -1,5 +1,3 @@
-// +build fuzz
```

Comment:
I had deliberately specified this build constraint. Could you revert this line?

Reply:
sorry, mis-comitted...